### PR TITLE
Add parameter for context in rbprm builder and fullbody

### PR DESCRIPTION
**Pull request by JasonChmn**
Add parameter to change context in constructors and function loadModel(...) of rbprmbuilder and rbprmfullbody.

### CODE REVIEW

File: src/hpp/corbaserver/rbprm/rbprmbuilder.py [hunk 1]
```diff
@@ -26,9 +26,12 @@
 #  trunk of the robot, and a set of robots describe the range of motion of each limb of the robot.
 class Builder(Robot):
     # # Constructor
-    def __init__(self, load=True):
+    def __init__(self, load=True, clientRbprm=None):
         self.tf_root = "base_link"
-        self.clientRbprm = RbprmClient()
+        if clientRbprm==None:
```
Comment (nim65s) [inline src/hpp/corbaserver/rbprm/rbprmbuilder.py]: `None` is a  singleton in Python, therefore you should use `if clientRbprm is None:`

**Review by nim65s — CHANGES_REQUESTED** [review]

File: src/hpp/corbaserver/rbprm/rbprmbuilder.py [hunk 2]
```diff
@@ -41,8 +44,8 @@ def __init__(self, load=True):
     # \param packageName name of the package from where the robot will be loaded
     # \param urdfSuffix optional suffix for the urdf of the robot package
     # \param srdfSuffix optional suffix for the srdf of the robot package
-    def loadModel(self, urdfName, urdfNameroms, rootJointType, meshPackageName, packageName, urdfSuffix, srdfSuffix):
-        Robot.__init__(self, urdfName, rootJointType, False)
+    def loadModel(self, urdfName, urdfNameroms, rootJointType, meshPackageName, packageName, urdfSuffix, srdfSuffix, client):
+        Robot.__init__(self, urdfName, rootJointType, False, client=client)
```
Comment (jmirabel) [inline src/hpp/corbaserver/rbprm/rbprmbuilder.py:48]: @stonneau @pFernbach Any idea why the parent initializer is called in loadModel and not in __init__ ?

Comment (pFernbach) [reply src/hpp/corbaserver/rbprm/rbprmbuilder.py:48]: If there was I don't remember. By the ways  shouldn't `client` have a default value to `None` to keep backward compatibility with the current scripts ?

File: src/hpp/corbaserver/rbprm/rbprmbuilder.py (hunk 1 above)
Comment (JasonChmn) [reply src/hpp/corbaserver/rbprm/rbprmbuilder.py]: Fix on syntax done in the last commit.

File: src/hpp/corbaserver/rbprm/rbprmbuilder.py (hunk 2 above)
Comment (JasonChmn) [reply src/hpp/corbaserver/rbprm/rbprmbuilder.py:48]: I set client parameter to None by default in loadModel() and loadFullBodyModel() in the last commit.

**Review by nim65s — APPROVED** [review]

**Comment by pFernbach** [comment]
Good for me.

**Pull request merged**